feature/metis/conversation/src/test/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ui/chatlist/MetisSearchPagingSourceTest.kt

```diff
@@ -0,0 +1,77 @@
+package de.tum.informatics.www1.artemis.native_app.feature.metis.conversation.ui.chatlist
+
+import androidx.paging.PagingSource
+import de.tum.informatics.www1.artemis.native_app.core.common.test.UnitTest
+import de.tum.informatics.www1.artemis.native_app.core.model.account.User
+import de.tum.informatics.www1.artemis.native_app.feature.metis.conversation.service.network.MetisService
+import de.tum.informatics.www1.artemis.native_app.feature.metis.shared.content.MetisContext
+import de.tum.informatics.www1.artemis.native_app.feature.metis.shared.content.dto.StandalonePost
+import de.tum.informatics.www1.artemis.native_app.feature.metistest.MetisServiceStub
+import kotlinx.coroutines.test.runTest
+import kotlinx.datetime.Clock
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.experimental.categories.Category
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+
+@Category(UnitTest::class)
+@RunWith(RobolectricTestRunner::class)
+class MetisSearchPagingSourceTest {
+
+    private lateinit var sut: MetisSearchPagingSource
+    private val metisServiceStub = MetisServiceStub()
+
+    @Before
+    fun setup() {
+        sut = MetisSearchPagingSource(
```

Comment:
There is no need to create the SUT in `before`, please just initialize it on the attribute.